build.gradle.kts

```diff
@@ -57,6 +57,13 @@ tasks.register("printPluginZipPath") {
     println(buildPlugin.outputs.files.first().path)
   }
 }
+tasks.register("printSignedPluginZipPath") {
+  doLast {
+    val signPlugin = tasks.findByPath(":signPlugin")!!
+    val signedPluginZipPath = signPlugin.outputs.files.first { file -> file.name.contains("signed") }.path
```

Comment:
Huh there are multiple files in the outputs? 🤔 what is the other file(s) that is/are filtered out?

Reply:
I'm not sure, what exactly are the outputs of this `signPlugin` task (it's description was not that clear to me). Let me test that with `first()` without this filtering.

Reply:
Ok, looks like we can go without filtering: 
```
Task :printSignedPluginZipPath
/root/git-machete-intellij-plugin/build/distributions/git-machete-intellij-plugin-3.0.3-6-SNAPSHOT+git.b63155a-signed.zip
```

Reply:
You can also use the very awesome `./gradlew :<task> :taskTree`; with some options it displays the inputs and/or outputs